src/algorithms/sorting/quick-sort/QuickSort.js

```diff
@@ -19,12 +19,12 @@ export default class QuickSort extends Sort {
     const rightArray = [];
 
     // Take the first element of array as a pivot.
-    const pivotElement = array.shift();
+    const pivotElement = array.pop();
```

Comment:
shift and pop are used for different purposes. shift removes element from the front of the array and pop removes from the end. hence pop cannot used to get the first element in the array.

Reply:
in this sutiation, it no matter the element is taken from the front of the array or the end

Reply:
@chandrunaik and @mrbeannnnn , in this case pop is better than shift because pop is O(1) (constant operation) where as shift() is O(n) .

Reply:
quick-sort should work well not metter the element is get from the top or from the end.
while in JS, pop is faster than shift, as talentedandrew said above.
so I think pop is better here.